dftimewolf/lib/module.py

```diff
@@ -19,25 +19,31 @@ class BaseModule(object):
     critical (bool): True if this module is critical to the execution of
         the recipe. If set to True, and the module fails to properly run,
         the recipe will be aborted.
+    name (str): A unique name for a specific instance of the module
+          class. If not provided, will default to the module's class name.
     state (DFTimewolfState): recipe state.
   """
 
-  def __init__(self, state, critical=False):
+  def __init__(self, state, name=None, critical=False):
     """Initialize a module.
 
     Args:
       state (DFTimewolfState): recipe state.
+      name (Optional[str]): The module's runtime name.
+      name (Optional[str]): A unique name for a specific instance of the module
```

Comment:
why is name twice here?

Reply:
I was overeager to document this new feature it seems.